code/src/Components/RadioButtonsWrapper.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import RadioButton from './RadioButton'
+
+const RadioButtonWrapper = (props) => {
+    return (
+        <form className="radio-buttons-wrapper">
+            <div className="button-label-wrapper">
+                <RadioButton 
+                    question={props.question}
+                    toChange={props.toChange}
+                    optionNumber={0}
+                    source={props.source}
+                />
+            </div>
+            <div className="button-label-wrapper">
+                <RadioButton 
+                    question={props.question}
+                    toChange={props.toChange}
+                    optionNumber={1}
+                    source={props.source}
+                />
+            </div>
+            <div className="button-label-wrapper">
+                <RadioButton 
+                    question={props.question}
+                    toChange={props.toChange}
+                    optionNumber={2}
+                    source={props.source}
+                />
+            </div>
+            <div className="button-label-wrapper">
+                <RadioButton 
+                    question={props.question}
+                    toChange={props.toChange}
+                    optionNumber={3}
+                    source={props.source}
+                />
+            </div>
+        </form>
+    )
+}
```

Comment:
I was wondering, whether you could think of a way of creating buttons with a map method rather than creating the same component over and over again. in you json file you have object called question that has an array of options. You can loop through that array and create radio button for each options.label. And of course each button should receive the same name but different ids.

Reply:
That is a great idea! Thank you ^^ will fix in a future commit